test/resources/ostree.resource

```diff
@@ -0,0 +1,62 @@
+*** Settings ***
+Documentation    Keywords for OSTree-based systems
+
+Library     SSHLibrary
+
+
+*** Variables ***
+${BACKUP_STORAGE}       /var/lib/microshift-backups
+${HEALTH_FILE}          /var/lib/microshift-backups/health.json
+
+
+*** Keywords ***
+Get Booted Deployment Status
+    [Documentation]    Get rpm-ostree status of booted deployment as a json
+
+    ${ostree_status}    ${rc}=    Execute Command
+    ...    rpm-ostree status --booted --json
+    ...    return_rc=True
+    Should Be Equal As Integers    0    ${rc}
+    ${json}=    Evaluate    json.loads('''${ostree_status}''')    json
```

Comment:
For YAML, I found it handy to parse the text and then return a RF DottedDict type to make accessing the data from RF keywords easier. See the YAML library in the resources directory.